modules/tinymce/src/core/main/ts/api/util/EventDispatcher.ts

```diff
@@ -296,7 +305,10 @@ class EventDispatcher<T> {
             // Unbind specific ones
             let hi = handlers.length;
             while (hi--) {
-              if (handlers[hi].func === callback) {
+              const handler = handlers[hi];
```

Comment:
[nitpick / optional] Theoretically, now that we have this `removed` flag, we could make this look more "functional"-ish, if you want to. Something like:

```typescript
Arr.forEach(handlers, (handler) => {
  if (handler.func === callback) {
    // do removing in two parts (flag the things to remove, and then remove them all later) to make iterating over a changing list simpler
    // not just here, but also if we happen to be in the middle of iterating over these handlers to fire this event right now
    handler.removed = true;
  }
});
this.bindings[currentName] = Arr.filter(handlers, (handler) => !handler.removed);
```

Reply:
Have changed this to be more functional, it's a bit different to what what was above though just to reduce the amount of looping being done.